src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -207,15 +208,23 @@ public class Session {
         }
 
         private async Task ClientToHostWorker(Stream stream, IMessagePipeEnd pipe) {
```

Comment:
>ClientToHostWorker [](start = 27, length = 18)

`ClientToHostWorker` doesn't create a stream, so I don't think it should dispose it either